submitted_models/marble_hd2_sensor_config_1/launch/vehicle_topics.launch

```diff
@@ -0,0 +1,207 @@
+      <remap from="$(arg sensor_prefix)/camera_front/points" to="front_facing_rgbd_camera/depth/points"/>
+    </node>
+    <node
+      pkg="ros_ign_image"
+      type="image_bridge"
+      name="ros_ign_image_depth_front"
+      args="$(arg sensor_prefix)/camera_front/depth_image">
+      <remap from="$(arg sensor_prefix)/camera_front/depth_image" to="front_facing/depth"/>
+    </node>
+
+    <!--30 Degree downward looking RGBD camera -->
+    <node
+      pkg="ros_ign_bridge"
+      type="parameter_bridge"
+      name="ros_ign_bridge_camera_down"
+      args="$(arg sensor_prefix)/camera_down/camera_info@sensor_msgs/CameraInfo[ignition.msgs.CameraInfo">
+      <remap from="$(arg sensor_prefix)/camera_down/camera_info" to="front_down/camera_info"/>
+    </node>
+    <node
+      pkg="ros_ign_image"
+      type="image_bridge"
+      name="ros_ign_image_camera_down"
+      args="$(arg sensor_prefix)/camera_down/image">
+      <remap from="$(arg sensor_prefix)/camera_down/image" to="front_down/image_raw"/>
+    </node>
+    <node
+      pkg="ros_ign_bridge"
+      type="parameter_bridge"
+      name="ros_ign_bridge_rgbd_down"
+      args="$(arg sensor_prefix)/camera_down/points@sensor_msgs/PointCloud2[ignition.msgs.PointCloudPacked">
+      <remap from="$(arg sensor_prefix)/camera_down/points" to="front_down_rgbd_camera/depth/points"/>
+    </node>
+    <node
+      pkg="ros_ign_image"
+      type="image_bridge"
+      name="ros_ign_image_depth_down"
+      args="$(arg sensor_prefix)/camera_down/depth_image">
+      <remap from="$(arg sensor_prefix)/camera_down/depth_image" to="front_down/depth"/>
+    </node>
+
+      <!--Pan Tilt RGBD camera -->
+    <node
+      pkg="ros_ign_bridge"
+      type="parameter_bridge"
+      name="ros_ign_bridge_camera_pan_tilt"
+      args="$(arg gimbal_sensor_prefix)/camera_pan_tilt/camera_info@sensor_msgs/CameraInfo[ignition.msgs.CameraInfo">
+      <remap from="$(arg gimbal_sensor_prefix)/camera_pan_tilt/camera_info" to="pan_tilt/camera_info"/>
+    </node>
+    <node
+      pkg="ros_ign_image"
+      type="image_bridge"
+      name="ros_ign_image_camera_pan_tilt"
+      args="$(arg gimbal_sensor_prefix)/camera_pan_tilt/image">
+      <remap from="$(arg gimbal_sensor_prefix)/camera_pan_tilt/image" to="pan_tilt/image_raw"/>
+    </node>
+    <node
+      pkg="ros_ign_bridge"
+      type="parameter_bridge"
+      name="ros_ign_bridge_rgbd_pan_tilt"
+      args="$(arg gimbal_sensor_prefix)/camera_pan_tilt/points@sensor_msgs/PointCloud2[ignition.msgs.PointCloudPacked">
+      <remap from="$(arg gimbal_sensor_prefix)/camera_pan_tilt/points" to="pan_tilt_rgbd_camera/depth/points"/>
+    </node>
+    <node
+      pkg="ros_ign_image"
+      type="image_bridge"
+      name="ros_ign_image_depth_pan_tilt"
+      args="$(arg gimbal_sensor_prefix)/camera_pan_tilt/depth_image">
+      <remap from="$(arg gimbal_sensor_prefix)/camera_pan_tilt/depth_image" to="pan_tilt/depth"/>
+    </node>
```

Comment:
here are the optical frame publisher nodes for `pan_tilt` rgbd camera:

```
    <node
      pkg="subt_ros"
      type="optical_frame_publisher"
      name="optical_frame_publisher_image_pan_tilt">
      <remap from="input/image" to="pan_tilt/image_raw" />
      <remap from="output/image" to="pan_tilt/optical/image_raw" />
      <remap from="input/camera_info" to="pan_tilt/camera_info" />
      <remap from="output/camera_info" to="pan_tilt/optical/camera_info" />
    </node>
    <node
      pkg="subt_ros"
      type="optical_frame_publisher"
      name="optical_frame_publisher_depth_pan_tilt"
      args="depth">
      <remap from="input/image" to="pan_tilt/depth" />
      <remap from="output/image" to="pan_tilt/optical/depth" />
    </node>
```

Reply:
Should I remove the following?
```
    <node
      pkg="ros_ign_image"
      type="image_bridge"
      name="ros_ign_image_depth_pan_tilt"
      args="$(arg gimbal_sensor_prefix)/camera_pan_tilt/depth_image">
      <remap from="$(arg gimbal_sensor_prefix)/camera_pan_tilt/depth_image" to="pan_tilt/depth"/>
    </node>
```